apps/meteor/server/startup/migrations/v303.ts

```diff
@@ -0,0 +1,19 @@
+import { Users } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+import { getNewUserRoles } from '../../services/user/lib/getNewUserRoles';
+
+addMigration({
+	version: 303,
+	name: 'Add default roles to users without roles field',
+	async up() {
+		const defaultRoles = getNewUserRoles();
+
+		await Users.updateMany(
+			{
+				roles: { $exists: false },
+			},
+			{ $set: { roles: defaultRoles } },
+		);
```

Comment:
IMO setting to just empty array is better here, goes with existing state and expectation.

Reply:
Yeah, that makes sense. I was thinking about a similar problem that we had with roles. But the best way to deal with this scenario is just to set an empty array of roles. The problem was only possible because the Default Role setting was set to empty.